app/presenters/cancellation_presenter.rb

```diff
@@ -29,6 +30,10 @@ def go_back_path
     referer_path || two_factor_options_path
   end
 
+  def url_options
+    @url_options
+  end
```

Comment:
I bet rubocop will yell, but this can just be an attr_reader

```suggestion
  attr_reader :url_options
```

Reply:
Good call, will change all of the ones where this pattern is used

Reply:
oops, realizing now that this will cause the lint to fail. It's less Ruby-ish to do this, but the explicitness may be helpful in troubleshooting `url_option` issues.